smartapps/konnected-io/konnected-security-connect.src/konnected-security-connect.groovy

```diff
@@ -127,11 +126,7 @@ def pageDiscovery() {
   //create accessToken
   if(!state.accessToken) { createAccessToken() }
 
-  //This is a workaround to prevent page to refresh too fast.
-  if(!state.pageConfigurationRefresh) { state.pageConfigurationRefresh = 2 }
-
-  dynamicPage(name: "pageDiscovery", nextPage: "pageConfiguration", refreshInterval: state.pageConfigurationRefresh) {
-    state.pageConfigurationRefresh =  state.pageConfigurationRefresh + 3
```

Comment:
I added back the backoff workaround for now at least. I'm not crazy about this workaround, but it's not too bad.